client/app/components/Form/DynamicForm.jsx

```diff
@@ -13,7 +13,7 @@ export type State = {
   formProps: any,
 };
 
-export class QuickCreateForm extends React.Component<Props, State> {
+export class DynamicForm extends React.Component<Props, State> {
```

Comment:
This form was originally used by QuickCreate (being able to create Categories/Moods/Strategies quickly within the Moments editor). I realized I could generalize this form and reuse it for forms in the frontend.